databases/week2/queryPart_3.sql

```diff
@@ -0,0 +1,25 @@
+-- Part 3: More queries
+-- * Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.* FROM task
+JOIN user ON task.user_id=user_id
+WHERE user.email= LIKE '%@spotify.com';
```

Comment:
What does this query(line 4 gives out..pls test it..?

Reply:
thats very interesting how it happened, i corrected